src/app/app.module.ts

```diff
@@ -3,6 +3,7 @@ import { NgModule, ErrorHandler } from '@angular/core';
 import { HttpModule } from '@angular/http';
 import { IonicApp, IonicModule, IonicErrorHandler } from 'ionic-angular';
 import { IonicStorageModule } from '@ionic/storage';
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
15 minutes of Googling turned up nothing, so I'm left with "because the changelog says so."  I might open a question on their forums.